helix-term/src/handlers/completion.rs

```diff
@@ -227,8 +227,8 @@ fn request_completion(
                 }
             };
 
-            let completion_response = ls.completion(doc_id, pos, None, context).unwrap();
-            async move {
+            let completion_response = ls.completion(doc_id, pos, None, context)?;
+            Some(async move {
```

Comment:
The `unwrap` is intentional here because `language_servers_with_feature(LanguageServerFeature::Completion)` should only give the language servers that will return Some from `completion`